notebooks/hive2bq/HiveToBigquery_notebook.ipynb

```diff
@@ -178,7 +183,8 @@
     "import time\n",
     "import os\n",
     "from pyspark.sql import SparkSession\n",
-    "import pandas as pd\n"
+    "import pandas as pd\n",
```

Comment:
This is not a new change. We are using pandas here so that we could save audit data as a csv file in GCS. As the number of records would not be alot, it didn't make sense to use Spark Dataframe here.